web/status-history/src/main/resources/application.yml

```diff
@@ -6,8 +6,10 @@ micronaut:
   router:
     static-resources:
       default:
+        enabled: true
```

Comment:
how does this help? would be nice to get references to the solutions used in the PR description